### ISSUE TITLE
Neovim+ALE: wrong squiggly location

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#771]
```js
let x = /*️⃣ I'm a comment! */ y;
console.log(x);
```
With Neovim and ALE and quick-lint-js LSP, E0057 is reported on the space between `!` and `*`. It should be on the letter `y` instead.

**Comment by strager** [comment]
Test case added to test/js/unicode.js in commit c838df7566a52f36b5500147aee221e3c7ea13bd.
